Cargo.toml

```diff
@@ -28,25 +30,27 @@ gloo-net = { version = "0.4", features = ["http"] }
 log = "0.4"
 cfg-if = "1.0"
 tracing = "0.1"
-reqwest = { version = "0.11", features = ["json"] }
 getrandom = { version = "0.2", features = ["js"] }
-lemmy_api_common = { tag = "0.19.0-rc.1", git = "https://github.com/LemmyNet/lemmy.git", default-features = false }
+lemmy_api_common = { git = "https://github.com/LemmyNet/lemmy.git", branch = "main", default-features = false }
```

Comment:
the version tag was good for not getting every single update from lemmy which is nice for stability. was there something broken or missing from lemmy for you?

Reply:
I was getting deserialization errors because the local_user on the site response had a field on it returned from the server that the current version of lemmy_api_common can't handle.

Reply:
ah i see. i guess that's because your local dev server is using the very latest version so you need them to be in sync.

i personally quite like the server api to be stable while i'm working against it.

i guess we can handle this locally.

Reply:
I do agree we should try to stay versioned, but ya I'll try to start another back-end beta deploy.